src/Command/PublicToInternalUriCommand.php

```diff
@@ -53,5 +53,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
             throw $exception;
         }
+
+        return Command::SUCCESS;
```

Comment:
Oja, dat is beter. Ik heb nu overal `return 0;` toegevoegd...

Moet dit trouwens niet even in een aparte PR? En ik zie geen toevoeging in de CHANGELOG hiervoor